Number of tags (1-5): Key #0: Value: Traceback (most recent call last):
  File "main.py", line 52, in <module>
    tagged_obj = SomeTaggableObject(**tags)
TypeError: __init__() got multiple values for argument 'self'
